src/actions/network.js

```diff
@@ -65,27 +72,24 @@ export function connectToNetwork (endpoint, tesseraEndpoint) {
  * successful and showing an error if unsuccessful.
  *
  * @param endpoint Geth RPC Url (http://localhost:22000)
- * @param tesseraEndpoint Tessera party keys endpoint (http://localhost:9081/partyinfo/keys)
+ * @param tesseraEndpoint Tessera endpoint (http://localhost:9081)
  * @returns thunk middleware dispatch function
  */
 export function saveNetwork (endpoint = '', tesseraEndpoint = '') {
   return async dispatch => {
     try {
-      // TODO clearing tessera endpoint every time since we are hiding that field for now. Delete this line later
-      tesseraEndpoint = ''
 
       if (tesseraEndpoint.endsWith('/')) {
         tesseraEndpoint = tesseraEndpoint.substring(0,
           tesseraEndpoint.length - 1)
       }
 
-      await testUrls(endpoint, tesseraEndpoint)
-
       // helper to automatically find partyinfo endpoint when adding a known local node
       if(endpoint.startsWith('http://localhost:2200')) {
         tesseraEndpoint = await getLocalPartyInfoIfAvailable(endpoint)
       }
```

Comment:
Let's get rid of this and the getLocalPartyInfoIfAvailable method below. It was just there because the tessera field was hidden.